content/en/05-reference/01-admin/11-user-mgmt.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "User Management & Sharing"
+metaTitle: "Reference | Admin | User Management & Sharing"
+metaDescription: "Share and manage access to your Pixie data."
+order: 11
+redirect_from:
+    - /admin/permissions/
+---
+
+Pixie has features to share and manage access to your Pixie organization.
+
+Instructions are based on identity provider. To determine your identity provider, open the Live UI sign in page (`/auth/login`):
+
+::: div image-m
+<svg title='Your Pixie uses Auth0 if you see the option to "Login with Google".' src='admin/user-mgmt/determine-auth.png'/>
+:::
+
+Follow the
+
+- [Auth0 Instructions](/reference/admin/user-mgmt/#auth0): if you have the option to `"Login With Google"
+- [Hydra / Kratos Instructions](/reference/admin/user-mgmt/#hydra-kratos): if you _do not_ have the option to `"Login With Google"
+
+## Hydra / Kratos
+
+Pixie instances using Hydra / Kratos as an identity provider can:
+
+- [Invite users](/reference/admin/user-mgmt#invite-users)
+- [Enable approvals for new users](/reference/admin/user-mgmt#enable-approvals-for-new-users)
+- [Remove users](/reference/admin/user-mgmt#enable-approvals-for-new-users)
+
+### Invite Users
+
+Users added to your organization can view live views, query running clusters, and deploy new Pixie clusters.
+
+1. Navigate to [`dev.withpixie.dev/admin/invite`](https://dev.withpixie.dev/admin/invite) in your browser.
+
+2. Fill out the necessary information, then click the `"Invite"` button.
+
+3. Copy and share the created link with your teammate. _Note that this link expires after 1 hour and cannot be recreated for the expired email address._
+
+<svg title='' src='admin/user-mgmt/os-auth-invite-link.png'/>
+
+### Enable Approvals for New Users
+
+Once enabled, every user who joins your org via an Invite Link must be manually approved before they are able to use Pixie.
+
+1. Navigate to [`dev.withpixie.dev/admin/org`](https://dev.withpixie.dev/admin/org) in your browser.
+
+2. Click `“Enable”` on the Approvals setting to enable Approvals. Note that if the button says `“Disable”`, this means User Approvals has already been enabled for your organization.
+
+<svg title='' src='admin/user-mgmt/os-enable-approvals.png'/>
+
+3. Manually approve new users under the Users tab ([`dev.withpixie.dev/admin/users`](https://dev.withpixie.dev/admin/users)). Once User Approvals is enabled, any new users that join your org will join in a disabled state. Click `“Approve”` on the appropriate user to allow that user complete access to your organization within Pixie.
+
+### Remove Users
```

Comment:
After discussing with Phillip & Vihang, it looks like enabling this for Hydra/Kratos isn't as simple as we originally expected... Removed users can currently end up in a bad state, so we won't be enabling this for Hydra/Kratos in the meantime. Lets remove this section.